bin/phpunit

```diff
@@ -1,13 +1,19 @@
 #!/usr/bin/env php
 <?php
 
-if (!file_exists(dirname(__DIR__).'/vendor/symfony/phpunit-bridge/bin/simple-phpunit.php')) {
-    echo "Unable to find the `simple-phpunit.php` script in `vendor/symfony/phpunit-bridge/bin/`.\n";
-    exit(1);
+if (!ini_get('date.timezone')) {
```

Comment:
Symfony now recommend now to use phpunit: https://github.com/symfony/recipes/pull/906

This has the advantage of not longer have hacky solutions for phpunit autoloading for phpstan and rector.